gems/bootstrap-sass/CVE-2016-10735.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2016-10735
+url: https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/
+title: XSS vulnerability in bootstrap-sass
```

Comment:
Perhaps add "via data-target" or something so it's obvious to differentiate vulns?